DSCResources/MSFT_xDnsServerForwarder/MSFT_xDnsServerForwarder.schema.mof

```diff
@@ -0,0 +1,7 @@
+[ClassVersion("1.0.0"), FriendlyName("xDnsServerForwarder")]
+class MSFT_xDnsServerForwarder :  OMI_BaseResource
+{
+    [Key] String IPAddress;
+    [Write] Boolean RemoveAll;
```

Comment:
I agree with @TravisEz13 - it might be better to turn this into a SingleInstance resource with the IPAddress parameter as an array.

Something like this:
```
[ClassVersion("1.0.0"), FriendlyName("xDnsServerForwarder")]
class MSFT_xDnsServerForwarder :  OMI_BaseResource
{
    [Key, Description("Specifies the resource is a single instance, the value must be 'Yes'"), ValueMap{"Yes"}, Values{"Yes"}] String IsSingleInstance;
    [Write] String IPAddress[];
}
```

This would do away with the need for the "RemoveAll" parameter and may even simplify the code (assuming the CIM query code can take an array of IPAddresses) :smile: